### PR TITLE
Bumps partytown version to fix deprecation warning

### LINKED ISSUE #5819: Partytown Deprecated API: pagespeed insights `Event.path` deprecation warning
### What version of `astro` are you using?

1.16.10

### Are you using an SSR adapter? If so, which one?

Netlify

### What package manager are you using?

yarn

### What operating system are you using?

Mac

### Describe the Bug

While using `@astrojs/partytown` integration, PageSpeed Insights warns about a deprecated API being used.

Screenshot 2023-01-10 at 18 47 05

The issue was fixed with this [PR](https://github.com/BuilderIO/partytown/pull/328) in `@builder.io/partytown` repo.

### Link to Minimal Reproducible Example

https://pagespeed.web.dev/report?url=https%3A%2F%2Fglass.no%2F

### Participation

- [X] I am willing to submit a pull request for this issue.

### PR DESCRIPTION
**Pull request by davlet61**
## Changes

This PR fixes #5819.

## Testing

No tests were added since it was only a version bump

## Docs

No need to update docs.

### CODE REVIEW

**Review by Princesseuh — APPROVED** [review]
Looks good. Thank you very much!

File: .changeset/fuzzy-phones-rule.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/partytown': minor
```
Comment (Princesseuh) [inline .changeset/fuzzy-phones-rule.md:2]: Oh actually, misread this. This can be a patch, the upstream package only had fixes

Comment (davlet61) [reply .changeset/fuzzy-phones-rule.md:2]: Oh yeah, makes sense. What should I do? :)

Comment (Princesseuh) [inline .changeset/fuzzy-phones-rule.md]:
```suggestion
'@astrojs/partytown': patch
```

Comment (davlet61) [reply .changeset/fuzzy-phones-rule.md]: Probably should've changed manually, ran the script instead
Sorry for the trouble :)

**Pull request merged**